metrics/grafana/tidb.json

```diff
@@ -16516,7 +16516,7 @@
             "y": 17
           },
           "hiddenSeries": false,
-          "id": 234,
+          "id": 299,
```

Comment:
```suggestion
          "id": 305,
```
The allocated id is increases to 304 after #42468, use 305 instead.

Reply:
I did consider the risk of conflict. Given 305 the current max, 299 is less than 305 and not chosen yet, isn't it more guaranteed to not be chosen by others?

Reply:
But 299 is already occupied.

https://github.com/pingcap/tidb/blob/9e8498510610d843268631f48b03b1f6280ce15d/metrics/grafana/tidb.json#L17432

Reply:
Thanks for the comments. Will use 305.

Reply:
Change done.